tools/src/main/java/org/apache/kafka/tools/ClientCompatibilityTest.java

```diff
@@ -261,20 +261,15 @@ void testAdminClient() throws Throwable {
                     nodes.size(), testConfig.numClusterNodes);
             }
             tryFeature("createTopics", testConfig.createTopicsSupported,
-                new Invoker() {
-                    @Override
-                    public void invoke() throws Throwable {
+                    () -> {
```

Comment:
Since `() -> {` is part of continuation in same statement(`tryFeature("createTopics", testConfig.createTopicsSupported, .. `), it gives 8 spaces for new line.

If the format is not as per project standard. could you guide me to the formatter xml we follow for Kafka? Or would you mind sharing yours'?

Reply:
We are not very consistent unfortunately. But I suggest sticking to the approach that was used previously in the file.

Reply:
ack